packages/framework/component-cache/jest.config.js

```diff
@@ -0,0 +1,2 @@
+const { configureReactNativeJest } = require('@fluentui-react-native/scripts');
+module.exports = configureReactNativeJest('android', { preset: 'react-native' });
```

Comment:
Jest recommends that you add the react-native preset for running in RN scenarios. I'm not actually sure how much this helps so I'm open to removing the preset.